org.hl7.fhir.validation/src/test/java/org/hl7/fhir/validation/tests/NativeHostServiceTester.java

```diff
@@ -13,7 +13,7 @@ public void test() throws Exception {
     
     NativeHostServices svc = new NativeHostServices();
     svc.init("hl7.fhir.r4.core#4.0.1");
-    svc.connectToTxSvc("http://tx.fhir.org/r4", null);
+    svc.connectToTxSvc("http://tx.fhir.org/r4", null, null);

```

Comment:
I really don't like this adding fields that we just expect the majority of users to null out. It's not clean at all.

We need to make other method signatures available for people to call if they don't want to pass a token.